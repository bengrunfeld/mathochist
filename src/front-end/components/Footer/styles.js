import styled from 'styled-components'

const FooterBar = styled.div`
  background: #101089;
  min-height: 150px;
  color: lightgray;
  padding-top: 20px;
`

export { FooterBar }
