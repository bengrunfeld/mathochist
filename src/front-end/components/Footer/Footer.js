import React from 'react'

import { FooterBar } from './styles'

const Footer = () => (
  <div className='container-fluid'>
    <FooterBar className='row'>
      <div className='col-12'>
        <h2>Footer</h2>
        <p>Other text</p>
      </div>
    </FooterBar>
  </div>
)

export default Footer
