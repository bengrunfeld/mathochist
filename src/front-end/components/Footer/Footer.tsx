import React from 'react'

import { FooterBar } from './styles'

const Footer = () => (
  <div className='container-fluid'>
    <FooterBar className='row'>
      <div className='col-6'>
        <p>Â© Ben Grunfeld 2019</p>
      </div>
    </FooterBar>
  </div>
)

export default Footer
