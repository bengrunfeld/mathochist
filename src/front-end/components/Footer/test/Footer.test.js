import React from 'react'
import Enzyme, { shallow, mount, render } from 'enzyme'
import Footer from '../Footer'

describe('<Footer />', () => {
  test('renders a p tag', () => {
    const component = mount(<Footer />)
    expect(component.find('p')).toHaveLength(1)
  })

  test('matches the snapshot', () => {
    const tree = mount(<Footer />)
    expect(tree).toMatchSnapshot()
  })
})
