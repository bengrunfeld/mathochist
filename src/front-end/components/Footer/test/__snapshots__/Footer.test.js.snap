// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches the snapshot 1`] = `
<Footer>
  <div
    className="container-fluid"
  >
    <styled.div
      className="row"
    >
      <StyledComponent
        className="row"
        forwardedComponent={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bdVaJa",
              "isStatic": true,
              "lastClassName": "iUiJFL",
              "rules": Array [
                "
  background: #0e0e58;
  min-height: 150px;
  color: lightgray;
  padding-top: 20px;
",
              ],
            },
            "displayName": "styled.div",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bdVaJa",
            "target": "div",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          }
        }
        forwardedRef={null}
      >
        <div
          className="row sc-bdVaJa iUiJFL"
        >
          <div
            className="col-6"
          >
            <p>
              Â© Ben Grunfeld 2019
            </p>
          </div>
        </div>
      </StyledComponent>
    </styled.div>
  </div>
</Footer>
`;
